contract-map.json

```diff
@@ -3727,5 +3727,12 @@
     "erc20": true,
     "symbol": "XTM",
     "decimals": 18
+  },
+  "0xae78736Cd615f374D3085123A210448E74Fc6393": {
+    "name": "Rocket Pool ETH",
+    "logo": "rETH.png",
```

Comment:
SCG is required

Reply:
Unfortunately the icon uses some complex gradients and subtle shadows which don't scale well using an SVG, which is why we've opted with a png (we don't have SVGs available because of the gradient scaling).

Is there any flexibility around using a png?

Reply:
@nickdoherty I think your logo can be coverted to SVG - take a look for example on https://github.com/MetaMask/contract-metadata/blob/b9aa76962fcbad6028e36cb1a8cdb31fd59f4274/images/COW.svg which is 2kB size vector image

Reply:
Unfortunately we've tried this a few times and we always end up with artifacts in the SVG as it scales (due to the multiple gradients and drop shadows). We can provide the png's at larger sizes if you need.

Reply:
I'm really sorry guys, but we have a strict policy of SVGs only..

Reply:
Have replaced with an SVG as requested.